CONTRIBUTING.md

```diff
@@ -73,7 +73,8 @@ To submit an update:
 
    Typically the previous command gives the URL to open a pull request. If not, you can open one from the [Pull requests](https://github.com/redhat-documentation/supplementary-style-guide/pulls) tab of the GitHub UI.
 
-After you submit a pull request, it will be reviewed by members of this project.
+After you submit a pull request, it will be reviewed by members of this project. 
+When the PR has been reviewed and approved by at least three `supplementary-style-guide` reviewers, the PR will be merged to the _main_ branch by a repository administrator.
```

Comment:
Should we specify that the PR must be approved by 3 repo admins, rather than any 3 SSG reviewers?